pkgs/development/compilers/gambit/default.nix

```diff
@@ -1,7 +1,8 @@
 { callPackage, fetchurl }:
 
-callPackage ./build.nix {
+callPackage ./build.nix rec {
   version = "4.8.9";
+  git-version = "v${version}";
```

Comment:
Defining `git-version` only makes sense if it is actually used by `build.nix`.

Reply:
Well,

1- I include it by symmetry with gerbil, that actually uses it. I actually intended to use it in gambit in the very same way, but that requires changes to the gambit build system similar to those done in Gerbil, that haven't made it on top of my todo list yet.

2- It's somewhat useful for me to track the progress of gambit unstable, and needed by symmetry in the stable build, too.

3- If you object to it, I can make it a comment instead, until I get to modifying the gambit build system (and get the changes upstreamed).

Reply:
Thank you for explaining. While I understand your intentions, unused code tends to confuse people and make packages harder to maintain - you might not be the only one modifying these  in the future. So please remove the unused code but feel free to add your comments.

Reply:
No problem. I understand your concerns. Here is an updated version where only the unstable build has the git-version, and only in a comment.